modules/aws-outposts-ingress-controller-service.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-ingress-controller-service_{context}"]
+= Configuring the default Ingress Controller in an AWS VPC cluster extended into an Outpost
+
+Classic Load Balancers in the AWS VPC cluster can attach to edge compute resources in the Outpost if edge and cloud-based subnets are in the same availability zone.
+As a result, Classic Load Balancers on the VPC cluster might schedule pods on either of these node types.
+
+The default Ingress Controller deployment on the VPC cluster is scheduled by a Classic Load Balancer.
+Because the load balancer can attach to edge compute resources in the Outpost, it might schedule Ingress Controller deployment on edge compute nodes.
+Scheduling on these nodes introduces latency between the router and the backend.
+To prevent the default Ingress Controller in the VPC cluster from attaching to Outpost edge compute nodes, you must apply labels to the cloud-based compute nodes and configure the default Ingress Controller to only schedule on nodes with the applied labels.
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Label the cloud-based compute nodes in the VPC cluster with a key-value pair by running the following command:
++
+[source,terminal]
+----
+$ for NODE in $(oc get node -l node-role.kubernetes.io/worker --no-headers | grep -v outposts | awk '{print$1}'); do oc label node $NODE <key_name>=<value>; done
```

Comment:
`grep` and `awk` are fine with me but I thought we try to limit the usage of the third party utils if there are `oc` equivalents. In this case we can use `-l` syntax:
```bash
$ NODES=$(oc get node -l '!node-role.kubernetes.io/outposts,node-role.kubernetes.io/worker' --template='{{range .items}}{{.metadata.name}} {{end}}')
$ oc label node ${NODES} label1=value1
```

or to have all on one line:
```bash
$ oc label node $(oc get node -l '!node-role.kubernetes.io/outposts,node-role.kubernetes.io/worker' --template='{{range .items}}{{.metadata.name}} {{end}}')  label1=value1
```

Reply:
yes - I often don't know how to form the appropriate `oc` command but def want to keep things as native to OCP as we can. Thank you!